ui/dashboards/src/components/PanelDrawer/PanelDrawer.tsx

```diff
@@ -68,16 +77,12 @@ const PanelDrawer = () => {
   // currently, we need to reset the states whenever panelDrawer is reopened
   // since this component does not get remounted when it open/closes (otherwise we lose the animation of sliding in/out)
   useEffect(() => {
-    setGroup(panelDrawer?.groupIndex);
-    if (panelDrawer?.panelKey) {
-      // display panel name and description in text fields when editing an existing panel
-      setPanelName(panels[panelDrawer.panelKey]?.spec.display.name ?? '');
-      setPanelDescription(panels[panelDrawer.panelKey]?.spec.display.description ?? '');
-    } else {
-      setPanelName('');
-      setPanelDescription('');
-    }
-  }, [panelDrawer, panels]);
+    setGroup(defaultGroup);
```

Comment:
Love that this got simpler. We can definitely still follow-up on trying to get rid of the effect completely.